apps/web/src/pages/ApplicantDashboardPage/components/CareerDevelopmentTaskCard.tsx

```diff
@@ -10,20 +10,50 @@ import {
   Well,
 } from "@gc-digital-talent/ui";
 import { commonMessages } from "@gc-digital-talent/i18n";
+import { empty, unpackMaybes } from "@gc-digital-talent/helpers";
 
 import useRoutes from "~/hooks/useRoutes";
+import FieldDisplay from "~/components/ToggleForm/FieldDisplay";
+import BoolCheckIcon from "~/components/BoolCheckIcon/BoolCheckIcon";
+import messages from "~/messages/careerDevelopmentMessages";
 
-import MoveInterestsList from "./MoveInterestsList";
-import OrganizationTypeInterestsList from "./OrganizationInterestsList";
 import FunctionalCommunityListItem from "./FunctionalCommunityListItem";
 
-const CareerDevelopmentTaskCard_Fragment = graphql(/* GraphQL */ `
+export const CareerDevelopmentTaskCard_Fragment = graphql(/* GraphQL */ `
   fragment CareerDevelopmentTaskCard on EmployeeProfile {
-    moveInterest {
+    lateralMoveInterest
+    lateralMoveTimeFrame {
       value
+      label {
+        en
+        fr
+        localized
+      }
```

Comment:
Oh, I missed this earlier. I don't think you need to query all 3 values, just localized should work since that is the only one being used

![image](https://github.com/user-attachments/assets/17927270-5672-46ee-adcc-57016423be93)

Reply:
Haha need to shake that habit off. Fixed in [Fix overfetching](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12841/commits/b02a280f8e24b51f6bf9f02bd2de6659c9910eb7).

Reply:
Oops, you kind of overdid it 😄 

![image](https://github.com/user-attachments/assets/8948a6a8-721f-4051-aee3-c6f3871730d4)

Reply:
Haha missed that error. Fixed in [Fix underfetching](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12841/commits/2a2185a80d75f89e6f14adcae417f02908e240a7)